inc/compat/plugins/compat-plugin-woocommerce-subscriptions.php

```diff
@@ -0,0 +1,198 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Compatibility with plugin: WooCommerce Subscriptions (by WooCommerce).
+ */
+class FluidCheckout_WooCommerceSubscriptions extends FluidCheckout {
+
+	/**
+	 * __construct function.
+	 */
+	public function __construct( ) {
+		$this->hooks();
+	}
+
+
+
+	/**
+	 * Initialize hooks.
+	 */
+	public function hooks() {
+		// Shipping methods section title
+		add_action( 'fc_after_shipping_method_options', array( $this, 'maybe_add_initial_shipment_title' ), 10 );
+
+		// Template file loader
+		add_filter( 'woocommerce_locate_template', array( $this, 'locate_template' ), 100, 3 );
+
+		// Shipping methods available for subscription plans
+		remove_action( 'woocommerce_subscriptions_recurring_totals_shipping', array( 'WCS_Template_Loader', 'get_recurring_cart_shipping' ), 10 );
+		add_action( 'fc_shipping_methods_after_packages_inside', array( 'WCS_Template_Loader', 'get_recurring_cart_shipping' ), 10 );
+	}
+
+
+
+	/**
+	 * Maybe add the initial shipment title to the shipping methods section.
+	 * 
+	 * @param  object  $method   The shipping method instance.
+	 */
+	public function maybe_add_initial_shipment_title( $package ) {
+		// Get packages count
+		$packages_count = $this->get_all_packages_count();
+
+		// Show initial shipment title when one-time product plus a subscription is selected or at least 2 subscription plans are selected
+		if ( $packages_count > 1 ) {
+			echo '<p class="woocommerce-shipping-contents"><small>' . __( 'Initial Shipment', 'woocommerce-subscriptions' ) . '</small></p>';
```

Comment:
@vlad-dedikov this is not semantically correct, since the text "Initial Shipment" is not the content of the shipping package.
Also, using the hook `fc_after_shipping_method_options` causes this text to be displayed multiple times when multi-package shipping is enabled by other plugins.